src/presentation/http/router/note.test.ts

```diff
@@ -377,32 +430,47 @@ describe('Note API', () => {
 
       body = await response?.json();
 
-      const expectedParentNote = {
-        'id': 55,
-        'publicId': 'Hu8Gsm0sA1',
-        'creatorId': 2,
-        'content': null,
-        'createdAt': '2023-10-16T13:49:19.000Z',
-        'updatedAt': '2023-10-16T13:49:19.000Z',
-      };
-
-      expect(body.parentNote).toStrictEqual(expectedParentNote);
+      expect(body.parentNote).toMatchObject({
+        creatorId: parentNote.creatorId,
+        id: parentNote.id,
+        publicId: parentNote.publicId,
+      });
     });
 
     test.todo('Returns 400 when parentId has incorrect characters and lenght');
   });
 
   describe('DELETE /note/:notePublicId', () => {
+    beforeEach(async () => {
+      /**
+       * Truncate all tables, which are needed
+       * Restart autoincrement sequences for data to start with id 1
+       *
+       * TODO get rid of restarting database data in tests (move to beforeEach)
+       */
+      await global.db.truncateTables();
+    });
     test('Returns 200 status and "true" if note was removed successfully', async () => {
-      const accessToken = global.auth(1);
-      const correctID = 'Pq1T9vc23Q';
+      /** Create test user */
+      const user = await global.db.insertUser({
```

Comment:
maybe put user creation in beforeAll in truncate to avoid creating a user every time?

Reply:
this seems clearer

Reply:
why?

Reply:
we will make global beforeEach( ) method for test to truncate data, then they would be doubled, that seems redundant to me, also arranging the data should be a part of the test i suppose